components/scim/org.wso2.carbon.identity.scim.provider/src/main/java/org/wso2/carbon/identity/scim/provider/impl/SCIMUserManager.java

```diff
@@ -959,16 +981,39 @@ public void deleteGroup(String groupId) throws NotFoundException, CharonExceptio
         }
     }
 
-    private User getSCIMUser(String userName) throws CharonException {
+    private User getSCIMMetaUser(String userName) throws CharonException{
+
+        List<String> claimURIList = new ArrayList<String>();
+        claimURIList.add(SCIMConstants.ID_URI);
+        claimURIList.add(SCIMConstants.META_LOCATION_URI);
+        claimURIList.add(SCIMConstants.META_CREATED_URI);
+        claimURIList.add(SCIMConstants.META_LAST_MODIFIED_URI);
+        User scimUser = null;
+
+        try {
+            Map<String, String> attributes = carbonUM.getUserClaimValues(
+                    userName, claimURIList.toArray(new String[claimURIList.size()]), null);
+            attributes.put(SCIMConstants.USER_NAME_URI, userName);
+            scimUser = (User) AttributeMapper.constructSCIMObjectFromAttributes(
+                    attributes, SCIMConstants.USER_INT);
+        } catch (UserStoreException e) {
+            throw new CharonException("Error in getting user information from Carbon User Store for " +
+                    "user: " + userName + " ", e);
+        } catch (CharonException e) {
+            throw new CharonException("Error in getting user information from Carbon User Store for " +
+                    "user: " + userName, e);
+        } catch (NotFoundException e) {
+            throw new CharonException("Error in getting user information from Carbon User Store for " +
+                    "user: " + userName, e);
+        }
```

Comment:
Thanks for the comment, this has already fixed with https://github.com/wso2/carbon-identity/commit/3a54764bf5ecf0126f17efc48646175fb77cc957